redis/src/main/scala/zio/redis/codec/StringUtf8Codec.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 John A. De Goes and the ZIO contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package zio.redis.codec
+
+import zio.redis.RedisError.CodecError
+import zio.schema.Schema
+import zio.schema.StandardType.{DoubleType, IntType, LongType}
+import zio.schema.codec.Codec
+import zio.stream.ZPipeline
+import zio.{Chunk, ZIO}
+
+import java.nio.charset.StandardCharsets
+
+private[redis] object StringUtf8Codec extends Codec {
```

Comment:
Why is it restored?

Reply:
cause i need light codec to execute commands in RedisClusterExecutorLive before connecting to cluster (request cluster configs, request ask command)